packages/palette/src/elements/Button/v3/Button.tsx

```diff
@@ -77,21 +77,19 @@ const Container = styled.button<ContainerProps>`
 
   ${boxMixin};
 
-  ${(props) => {
-    // Handle sizing
-    return variant({ prop: "size", variants: BUTTON_SIZES })(props)
-  }};
+  /* Handle sizing */
+  ${variant({ prop: "size", variants: BUTTON_SIZES })}
 
   ${(props) => {
     // Handle props driven states
     if (props.hover) {
-      return variant({ variants: BUTTON_VARIANTS.hover })(props)
+      return variant({ variants: BUTTON_VARIANTS.hover })
```

Comment:
Accidentally realized you don't have to thread through the props for these variant calls. Likely for situations exactly like this.